src/monitoring_tests/cost_coverage_zero_signed_fee.py

```diff
@@ -63,19 +54,16 @@ def cost_coverage(self, competition_data: dict[str, Any], gas_cost: float) -> bo
                 / 10**36
             )
             total_fee += fee
-        if total_fee - gas_cost > 0.002 or total_fee - gas_cost < -0.002:
-            if zero_signed_fee_market and (
-                total_fee - gas_cost > 0.02 or total_fee - gas_cost < -0.01
-            ):
-                self.alert(
-                    f'"Fees - gasCost" is {total_fee - gas_cost} \
-                        for {competition_data["transactionHash"]}.'
-                )
-            else:
-                self.logger.info(
-                    f'"Fees - gasCost" is {total_fee - gas_cost} \
+        if total_fee - gas_cost > 0.02:
+            self.alert(
+                f'"Fees - gasCost" is {total_fee - gas_cost} \
                     for {competition_data["transactionHash"]}.'
-                )
+            )
+        elif total_fee - gas_cost < -0.04:
+            self.logger.info(
+                f'"Fees - gasCost" is {total_fee - gas_cost} \
+                for {competition_data["transactionHash"]}.'
+            )
```

Comment:
This log message is quite different from what other tests emit. It also does not follow the `log_output = ...` and `self.alert(log_output)` convention.